dashboard/modules/node/node_head.py

```diff
@@ -235,14 +244,49 @@ async def get_node_module_internal_state(self, req) -> aiohttp.web.Response:
             **self.get_internal_states(),
         )
 
+    @routes.get("/nodes/logical_resource")
```

Comment:
```suggestion
    @routes.get("/nodes/logical_resources")
```

Can you answer @rkooo567 question on why this needs to be a separate endpoint?

Reply:
Sure, Here is my previous comment:
@alanwguo @rkooo567 

> 
> Creating a new endpoint would adhere to the single responsibility principle.
> Easy to maintain and test
> Currently, api/cluster_status serves to fetch cluster autoscaler information. We use format=0 to retrieve JSON data and format=1 to get a formatted string. For the logical resource info, we could consider adding another query parameter like “api/cluster_status?query=logical_resource".
> 
> However, I believe that /nodes/logical_resource is simple enough and would be a better option. This endpoint would return a dictionary containing {node_id: logical_resource_str}, making it easy to maintain and test without modifying the existing endpoint